### PR TITLE
Ports pools from NSV13

### PR DESCRIPTION
**Pull request by maxion12345**
todo
- [ ] snip
snip
- [x] Throw people into pools
currently the pool acts like an invisible wall and you will be stopped
# Document the changes in your pull request
Port of https://github.com/BeeStation/BeeStation-Hornet/pull/3160

# Wiki Documentation
update photo

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  PowerfulBacon, KMC, Cdey, Maxim
rscadd: Added pool to boxstation
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
felinid interactions actually pretty funny

**Comment by KillerOrcaCora** [comment]
if you need custom feline bikini sprites, just give the word. (Very serious)

**Comment by maxion12345** [comment]
linter green mmm yes

**Comment by UselessTheremin** [comment]
> if you need custom feline bikini sprites, just give the word. (Very serious)

main question is why the heck would they need a different lookin bikini they are humans with ears and a tail slapped on

**Comment by maxion12345** [comment]
i've got work for the next few days ergo this will remain stagnant for now

especially because I can't seem to understand how shove/throwing works and I'll need time to bang my head against the proverbial wall in order to complete this pr

**Review by nesisknesis — APPROVED** [review]

**Comment by KillerOrcaCora** [comment]
> > if you need custom feline bikini sprites, just give the word. (Very serious)
> 
> main question is why the heck would they need a different lookin bikini they are humans with ears and a tail slapped on

Might as well add to the list of felind only equipable drip wont be the first or the last.

**Comment by maxion12345** [comment]
declaring this as ready, shoving into pools wasn't feasible (I couldn't figure out how to make it work)

you can throw people into pools though, so maybe its better from a balance perspective.

**Review by nesisknesis — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Comment by JamieD1** [comment]
@wejengin2

**Review by patpol4 — APPROVED** [review]
It's only on box, which is fine for now, might be worth testmerging this

File: code/modules/pool/pool.dm [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+	user.forceMove(src)
+	playsound(src, 'sound/effects/splosh.ogg', 100, 1) //Credit to hippiestation for this sound file!
+	user.visible_message("<span class='boldwarning'>SPLASH!</span>")
+	var/zap = 0
+	if(issilicon(user)) //Do not throw brick in a pool. Brick begs.
+		zap = 1 //Sorry borgs! Swimming will come at a cost.
+	if(ishuman(user))
+		var/mob/living/carbon/human/F = user
+		var/datum/species/SS = F.dna.species
+		if(MOB_ROBOTIC in SS.inherent_biotypes)  //ZAP goes the IPC!
```
Comment (Xoxeyos) [inline code/modules/pool/pool.dm]: We don't have IPCs.

**Comment by JamieD1** [comment]
Fix cornflakes

**Comment by maxion12345** [comment]
NO

**Comment by maxion12345** [comment]
NOT AGAIN

**Comment by maxion12345** [comment]
the fucking horror files refuse to go away, i cant discard them at all its extremely fucking annoying

**Comment by maxion12345** [comment]
pools closed for now these fucking horror changes refuse to leave so I cannot actually fix the merge conflict
I'll probably end up making a new branc- oh, I cant, because the changes are there greattt

**Pull request closed without merging**